thirdparty/scripts/build_arrow.sh

```diff
@@ -35,9 +35,12 @@ else
 fi
 
 # Download and compile arrow if it isn't already present.
-if [[ ! -d $TP_DIR/pkg/arrow ]]; then
+if [[ ! -d $TP_DIR/../python/ray/pyarrow_files/pyarrow ]]; then
     echo "building arrow"
-    git clone https://github.com/apache/arrow.git "$TP_DIR/build/arrow"
+
+    if [[ ! -d $TP_DIR/build/arrow ]]; then
+      git clone https://github.com/apache/arrow.git "$TP_DIR/build/arrow"
```

Comment:
this doesn't fail the build right?

Reply:
no it doesn't, the build should fail only if `make VERBOSE=1 -j$PARALLEL` below fails, I think

Reply:
wait is it supposed to? what is the name of the PR referring to?

Reply:
You're right, the name is wrong, I'll fix it.